core/src/main/resources/log4j4ibis.xml

```diff
@@ -98,7 +98,11 @@
 		<Logger name="org.quartz" level="WARN" />
 		<Logger name="nl.nn.adapterframework.configuration.digester" level="WARN" />
 
-		
+		<Logger name="liquibase.migrationLog" level="INFO">
+			<AppenderRef ref="stdout"/>
+			<AppenderRef ref="config"/>
```

Comment:
Ik denk dat het handiger/prettiger is om een losse 'databasemigration.log' file te hebben.
a) om de database structuur historie makkelijk terug te zien
b) om het zoeken in config.log niet te bemoeilijken